src/Module/Item/Compose.php

```diff
@@ -205,6 +207,7 @@ protected function content(array $request = []): string
 				'location_disabled'    => $this->l10n->t('Location services are disabled. Please check the website\'s permissions on your device'),
 				'wait'                 => $this->l10n->t('Please wait'),
 				'placeholdertitle'     => $this->l10n->t('Set title'),
+				'placeholdersummary'   => Feature::isEnabled(DI::userSession()->getLocalUserId(), 'summary') ? $this->l10n->t('Set summary') : '',
```

Comment:
This placeholder isn't helpful if the "Summary" field ends up being used as content warning.